incubator/azuremonitor-containers/templates/omsagent-rbac.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     heritage: {{ .Release.Service }}
 ---
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Why are you changing this back to no stable API versions?

Reply:
Update PR comments. We will update the API version once subsequent PR along with changes to add the support for 1.16 version.  reverting API version to extensions/v1beta1 to get the chart working since this change API version change broke the chart.